### PR TITLE
Fix #176

### LINKED ISSUE #176: [settings] add header

### PR DESCRIPTION
**Pull request by AboShanab**

### CODE REVIEW

File: App/screens/SettingsScreen.js [hunk 1]
```diff
@@ -9,6 +9,22 @@ import { clearCache } from '../../utils';
 import MutedText from '../components/MutedText';
 
 class SettingsScreen extends React.Component {
+  static options() {
+    return {
+      topBar: {
+        visible: true,
+        drawBehind: false,
+        backButton: {
+          color: 'black'
```
Comment (generalpiston) [inline App/screens/SettingsScreen.js]: Kick:
```
backButton: {
      visible: false
    },
```

File: App/screens/SettingsScreen.js [hunk 2]
```diff
@@ -9,6 +9,22 @@ import { clearCache } from '../../utils';
...
+        },
+        title: {
+          text: 'Mobidex Settings',
```
Comment (generalpiston) [inline App/screens/SettingsScreen.js]: Change to `Settings` or `About`

Comment (AboShanab) [reply App/screens/SettingsScreen.js]: Done.

File: App/screens/SettingsScreen.js (hunk 1 above)
Comment (AboShanab) [reply App/screens/SettingsScreen.js]: Done.

**Pull request merged**